daemon/cmd/daemon_main.go

```diff
@@ -280,6 +280,7 @@ func InitGlobalFlags(cmd *cobra.Command, vp *viper.Viper) {
 	option.BindEnv(vp, option.IPv6MCastDevice)
 
 	flags.Bool(option.EnableRemoteNodeIdentity, defaults.EnableRemoteNodeIdentity, "Enable use of remote node identity")
+	flags.MarkDeprecated(option.EnableRemoteNodeIdentity, "This option will be removed in v1.16 and will be \"true\" by default.")
```

Comment:
I'd slightly rephrase the end of the message, as  to me  'true by default' would imply that this setting is still configurable. The message in the PR description looks clearer.